hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    createAsyncThunk,
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+    PayloadAction,
+} from "@reduxjs/toolkit";
+import { Order } from "api/types";
+import { get } from "api/api";
+import { RootState } from "../store";
+import { cartReducerName } from "../hardware/cartSlice";
+
+interface OrderExtraState {
+    isLoading: boolean;
+    error: string | null;
+}
+
+const extraState: OrderExtraState = {
+    isLoading: false,
+    error: null,
+};
+
+const orderAdapter = createEntityAdapter<Order>({
+    selectId: (entity) => entity.id,
```

Comment:
this is not needed because by default, selectId selects entity.id so you can remove this argument: `createEntityAdapter<Order>()`